package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frint",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
please don't update the version numbers in PRs. we do it via `npm version` while releasing separately :)